mailer.js

```diff
@@ -26,6 +27,28 @@ module.exports = function (log) {
     return 'href="' + url + '" style="color: #0095dd; text-decoration: none; font-family: sans-serif;"'
   }
 
+  function constructLocationString (location) {
+    // construct the location string from the location object
+    var locationString = ''
+    if (location !== null) {
+      if (location.city) {
+        locationString += location.city + ', '
+      }
+      locationString += location.country + ' ' + '(estimated)'
```

Comment:
https://github.com/mozilla/fxa-auth-mailer/search?utf8=%E2%9C%93&q=gettext

I think it should be `gettext` or something, check others